src/js/jsx/shared/Datalist.jsx

```diff
@@ -451,6 +475,15 @@ define(function (require, exports, module) {
             }
         },
 
+        removeAutofillSuggestion: function () {
+            if (this.props.useAutofill) {
+                this.setState({
+                    suggestTitle: ""
+                });
+                this.forceUpdate();
```

Comment:
The `setState` call above will cause `render` to be run assuming the state passes through `shouldComponentUpdate`. Why is the extra `forceUpdate` call necessary?